spec/integration/integration_spec.rb

```diff
@@ -16,15 +16,12 @@
         @fake_gluster = stub_gluster(@peer_provider)
       end
 
-      describe 'gluster_peer' do
+      describe 'gluster_peer create' do
         it 'should add a single peer (manifest)' do
           expect(@fake_gluster.peer_hosts).to eq([])
-          x = apply_manifest(<<-'END'
-            node default {
-              gluster_peer { 'gfs1.local': }
-            }
-            END
-          )
+          x = apply_node_manifest(<<-'MANIFEST')
```

Comment:
The content of the heredoc only starts on the line after the starting syntax, and that syntax is replaced by the content. Thus, you can treat the `<<-'END'` as if it were a string literal containing the content. (For example, you can do `foo = <<-'END'.gsub(/re/, repl)` followed by the lines of content.) The end token must be the only non-whitespace on its line, so you can't do `END )` or whatever.

I usually find that sort of thing hard to read, but in this case the `MANIFEST` serves as an end-of-content indicator and the extra line for the closing paren just wastes vertical space.